conanfile.py

```diff
@@ -1,23 +1,34 @@
-from conans import ConanFile
+from conans import ConanFile, tools
```

Comment:
I think the `tools` import here can be removed.

Reply:
:+1: copypasta